File: Hacktoberfest-2023/Leetcode Solutions/Problem 1512.java
class Solution {
    public int numIdenticalPairs(int[] nums) {
        int count=0;
        int i,j;
        for(i=0;i<nums.length;i++){
            for(j=i+1;j<nums.length;j++){
                if(nums[i]==nums[j]){
                    count++;
                }
            }
        }
        return count;
    }
}

File: Hacktoberfest-2023/Leetcode Solutions/Problem 1512.py

class Solution:
    def numIdenticalPairs(self, nums: List[int]) -> int:
        count = 0

        for i in range(len(nums)):
            for j in range(len(nums)):
                if i<j and nums[i] == nums[j]:
                    count += 1
                    
        return count  
